pkg/scheduler/api/node_info.go

```diff
@@ -283,7 +283,7 @@ func (ni *NodeInfo) RemoveTask(ti *TaskInfo) error {
 			ni.Releasing.Sub(task.Resreq)
 			ni.Idle.Add(task.Resreq)
 			ni.Used.Sub(task.Resreq)
-			ni.AddGPUResource(ti.Pod)
+			ni.SubGPUResource(ti.Pod)
```

Comment:
Why decreace GPU resource?

Reply:
Because it's removing task from node, how could we add resource here?